src/Shared/ISessionDataSerializer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System.Web.SessionState;
+
+namespace Microsoft.Web.RedisSessionStateProvider
+{
+    public interface ISessionDataSerializer
```

Comment:
Interface members should all have doc comments

Reply:
added